File: brightcove-exoplayer/AppCompatSampleApp/src/main/java/com/brightcove/player/samples/appcompat/FragmentPlayerActivity.java
package com.brightcove.player.samples.appcompat;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

/**
 * Hosts the {@link PlayerFragment}, demonstrating the AppCompat plugin with a
 * BrightcovePlayerFragment.
 */
public class FragmentPlayerActivity extends AppCompatActivity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_fragment);
    }
}

File: brightcove-exoplayer-kotlin/AppCompatSampleApp/src/main/java/com/brightcove/appcompat/FragmentPlayerActivity.kt
package com.brightcove.appcompat

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

/**
 * Hosts the [PlayerFragment], demonstrating the AppCompat plugin with a BrightcovePlayerFragment.
 */
class FragmentPlayerActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_fragment)
    }
}
